dc-model/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>de.digitalcollections.model</groupId>
     <artifactId>dc-model-parent</artifactId>
-    <version>1.1.0-SNAPSHOT</version>
+    <version>2.0.0-SNAPSHOT</version>
```

Comment:
hier warnung "ist hardgecoded im jackson-module" o.ä.?

Reply:
Externalized to properties file: solved by https://github.com/dbmdz/digitalcollections-model/commit/94da371b44ca7f38888784a4fd6e6398d0eee811